pkg/secrethub/credentials/key.go

```diff
@@ -65,9 +65,17 @@ func ImportKey(credentialReader, passphraseReader Reader) (Key, error) {
 			return Key{}, errors.New("need passphrase")
 		}
 
+		// Try up to three times to get the correct passphrase.
 		for i := 0; i < 3; i++ {
-			credential, err := decryptKey(passphraseReader, encoded)
+			passphrase, err := passphraseReader.Read()
+			if err != nil {
+				return Key{}, err
+			}
+			if len(passphrase) == 0 {
+				return Key{}, errors.New("no passphrase given")
```

Comment:
It used to retry when you don't provide a passphrase, now it exits. Is that an intentional change in behavior?

Reply:
It was not an intentional change, but it is intentional behaviour. Because I did not see much purpose in trying again if you do not enter anything.

But we discussed it here and conclude that it might be a bit more user-friendly to retry.

Reply:
Added, see https://github.com/secrethub/secrethub-go/pull/117/commits/737803b2fb291293bafd01aa0e1a338fef4893ab